package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "libvirt",
-  "version": "1.0.0",
+  "version": "1.1.0",
```

Comment:
I wonder if people using 0.10.x has been using this module. If so, I would change the major number instead, given that we are dropping support for that version and it will be a breaking change for those users.